src/components/macroview/rightPanel/editForms/OpenEventForm.tsx

```diff
@@ -40,7 +52,29 @@ export default function OpenEventForm({
         break
     }
     setHeaderText(
-      sysEventLookup.get(selectedElement.data.action.type)?.displayString || ''
+      <HStack justifyContent="center" p={1}>
+        <Text>Editing element</Text>
+        <Box
+          h="32px"
+          w="fit-content"
+          bg={bg}
+          border="1px solid"
+          py={1}
+          px={3}
+          borderColor={kebabColour}
+          rounded={borderRadiusStandard}
+        >
+          <Text
+            w="fit-content"
+            whiteSpace="nowrap"
+            fontSize={['sm', 'md', 'md']}
+            fontWeight="bold"
+          >
+            {sysEventLookup.get(selectedElement.data.action.type)
+              ?.displayString || ''}
+          </Text>
+        </Box>
+      </HStack>
     )
```

Comment:
Putting a whole component into a state variable is really strange and should only be used if you have no other nice option

Reply:
Tweaked: [1608261](https://github.com/WootingKb/wooting-macros/pull/194/commits/1608261629ba0b9b89c1fb38fd90dcce0a3da03f)